log.c:174:5: error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
--- log.c.orig	2000-06-22 17:55:13.000000000 -0500
+++ log.c	2021-02-14 04:22:11.000000000 -0600
@@ -38,6 +38,7 @@
 #include <syslog.h>
 #include <stdio.h>
 #include <stdarg.h>
+#include <stdlib.h>
 #include <errno.h>
 #include <signal.h>
 #include <sys/types.h>
